packages/react-ui/src/theme/sections/typography.js

```diff
@@ -111,6 +111,10 @@ const customTypography = {
     ...baseTypography.body1,
     fontStyle: 'italic'
   },
+  body1Medium: {
```

Comment:
I'm going to remove most of them in the branch with the new Typography component.